frontend/src/components/widgets/FileUploader/styled-components.ts

```diff
@@ -0,0 +1,164 @@
+import styled, { CSSObject } from "@emotion/styled"
+import { Theme } from "theme"
+
+export interface StyledFileDropzone {
+  isDisabled: boolean
+}
+
+export const StyledFileDropzoneSection = styled.section<StyledFileDropzone>(
+  ({ isDisabled, theme }) => ({
+    display: "flex",
+    alignItems: "center",
+    padding: theme.spacing.lg,
+    backgroundColor: theme.inSidebar
+      ? theme.colors.white
+      : theme.colors.lightestGray,
+    borderRadius: theme.radii.md,
+    ":focus": {
+      outline: "none",
+      boxShadow: `0 0 0 1px ${theme.colors.primary}`,
+    },
+    color: isDisabled ? theme.colors.gray : theme.colors.bodyText,
+  })
+)
+
+export const StyledFileDropzoneButtonContainer = styled.span({})
+
+export const StyledFileDropzoneInstructions = styled.div(({ theme }) => ({
+  marginRight: "auto",
+  alignItems: "center",
+  display: "flex",
+}))
+
+export const StyledFileDropzoneInstructionsFileUploaderIcon = styled.span(
+  ({ theme }) => ({
+    color: theme.colors.secondary,
+    marginRight: theme.spacing.lg,
+  })
+)
+
+export const StyledFileDropzoneInstructionsStyledSpan = styled.span(
+  ({ theme }) => ({
+    marginBottom: theme.spacing.twoXS,
+  })
+)
+
+export const StyledFileDropzoneInstructionsColumn = styled.div({
+  display: "flex",
+  flexDirection: "column",
+})
+
+export const StyledUploadedFiles = styled.div(({ theme }) => ({
+  left: 0,
+  right: 0,
+  lineHeight: theme.lineHeights.tight,
+  paddingTop: theme.spacing.md,
+  paddingLeft: theme.spacing.lg,
+  paddingRight: theme.spacing.lg,
+}))
+
+export const StyledUploadedFilesList = styled.ul(({ theme }) => ({
+  listStyleType: "none",
+  margin: theme.spacing.none,
+}))
+
+export const StyledUploadedFilesListItem = styled.li(({ theme }) => ({
+  margin: theme.spacing.none,
+  padding: theme.spacing.none,
+}))
+
+export const StyledUploadedFileData = styled.div(({ theme }) => ({
+  display: "flex",
+  alignItems: "baseline",
+  flex: 1,
+  paddingLeft: theme.spacing.lg,
+  overflow: "hidden",
+}))
+
+export const StyledUploadedFileName = styled.div(({ theme }) => ({
+  marginRight: theme.spacing.sm,
+  marginBottom: theme.spacing.twoXS,
+  overflow: "hidden",
+  textOverflow: "ellipsis",
+  whiteSpace: "nowrap",
+}))
+
+export const StyledUploadedFile = styled.div(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  marginBottom: theme.spacing.twoXS,
+}))
+
+export const StyledErrorMessage = styled.span(({ theme }) => ({
+  marginRight: theme.spacing.twoXS,
+}))
+
+export const StyledFileIcon = styled.div(({ theme }) => ({
+  display: "flex",
+  padding: theme.spacing.twoXS,
+  color: theme.colors.secondary,
+}))
+
+export const StyledFileError = styled.small(({ theme }) => ({
+  color: theme.colors.danger,
+  fontSize: theme.fontSizes.smDefault,
+  height: theme.fontSizes.smDefault,
+  lineHeight: theme.fontSizes.smDefault,
+  display: "flex",
+  alignItems: "center",
```

Comment:
I think if we leave it as Small component we can remove all this?

Reply:
The problem was we targeted the `small` tag, so we can import the Small component and have the styled components change these.

In reality, we should create a generic `Text` component to allow some customization.